src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/generated.py

```diff
@@ -41,8 +41,8 @@
 cli_command('storage account list', list_storage_accounts)
 cli_command('storage account show-usage', show_storage_account_usage)
 cli_command('storage account update', set_storage_account_properties)
-cli_command('storage account connection-string', show_storage_account_connection_string)
-cli_command('storage account keys renew', renew_storage_account_keys)
+cli_command('storage account get-connection-string', show_storage_account_connection_string)
```

Comment:
It's even the name of the method :)

---
In reply to: [78055241](https://github.com/Azure/azure-cli/pull/843#discussion_r78055241) [](ancestors = 78055241)